package/nativeComponents/layouts/NativeList.js

```diff
@@ -1,8 +1,289 @@
 // eslint-disable-next-line no-unused-vars, unused-imports/no-unused-imports
 import React from "react";
 
+// eslint-disable-next-line import/no-unresolved
+import { CoreBox } from "@wrappid/core";
+// eslint-disable-next-line import/no-unresolved
+import { UtilityClasses } from "@wrappid/styles";
+
 import { SCList } from "../../styledComponents/layouts/SCList";
 
 export default function NativeList(props) {
-  return <SCList {...props}>{props.children}</SCList>;
+  const { children, ...restProps } = props; 
+  const listStyleClasses = () => {
+    let styleClasses = [];
+
+    styleClasses.push(UtilityClasses.PADDING.PL5);
+    switch (props.listType) {
```

Comment:
What if we can remove this switch case with the below statement 

```
styleClasses.push(UtilityClasses.LIST_STYLE[props.listType];
```

Reply:
ok thats a great idea i will try to implement it.